### ISSUE TITLE
Action head gradients are not synchronized in multi-GPU training because DDP forward is bypassed

### ISSUE THREAD

**Issue by TJ12342** [moojink/openvla-oft#160]
During multi-GPU fine-tuning, the action head is wrapped with `DistributedDataParallel`, but the training code calls methods directly on `action_head.module`:

https://github.com/moojink/openvla-oft/blob/main/vla-scripts/finetune.py#L388-L394

```python
if use_l1_regression:
    predicted_actions = action_head.module.predict_action(actions_hidden_states)

if use_diffusion:
    noise_pred = action_head.module.predict_noise(actions_hidden_states)
```

Calling the underlying `.module` bypasses `DistributedDataParallel.forward()`. As a result, the DDP reducer is not prepared for the backward pass and the action-head gradients are not all-reduced across ranks.

This is different from synchronizing the scalar loss value. The loss itself does not need to be all-reduced, but gradients of the DDP-wrapped action head should be.

## Impact

Each rank updates its own action head using only its local batch:

```text
rank 0 action head <- rank 0 local batch
rank 1 action head <- rank 1 local batch
...
```

Consequently:

- action-head parameters diverge across ranks;
- the action head effectively trains with the local batch size instead of the global batch size;
- the VLA backbone can still receive synchronized DDP gradients, but those gradients are computed through different rank-local action heads;
- checkpoint saving on the main process preserves only rank 0's version of the action head.

This affects both L1 regression and diffusion training. Single-GPU training is not affected in the same way.
